.github/workflows/ci.yaml

```diff
@@ -81,7 +82,8 @@ jobs:
     name: ${{ matrix.tox-environments }} | ${{ matrix.ubuntu-versions.series }}
     needs:
       - lint
-      - unit-test
+      # TODO: re-enable after adding unit tests
```

Comment:
Just to be sure, please drop a ticket to the next pulse / we need to invest here ASAP.